custom/icds/messaging/indicators.py

```diff
@@ -115,8 +140,6 @@ class LSSubmissionPerformanceIndicator(LSIndicator):
     def __init__(self, domain, user):
         super(LSSubmissionPerformanceIndicator, self).__init__(domain, user)
 
-        child_locations = [l.location_id for l in self.user.sql_location.child_locations()]
-        self.aww_user_ids = user_ids_at_locations(child_locations)
```

Comment:
That would require way more queries though (20x I think?)

Curious why you prefer couch vs es? If we have inconsistencies (especially for things that are mostly static like users and locations) I think that would be a much larger issue.